backend/infrahub/message_bus/operations/git/repository.py

```diff
@@ -44,3 +45,31 @@ async def import_objects(message: messages.GitRepositoryImportObjects, service:
         )
         repo.task_report = git_report
         await repo.import_objects_from_files(infrahub_branch_name=message.infrahub_branch_name, commit=message.commit)
+
+
+@flow(name="refresh-git-fetch")
```

Comment:
Would be good to add a flow_run_name
flow_run_name="Fetch git repository {message.repository_name} on " + WORKER_IDENTITY